@AH-64D Apache Longbow/Addons/fza_ah64_controls/scripting/functions/hellfire/fn_getAttackProfile.sqf

```diff
@@ -0,0 +1,35 @@
+/* ----------------------------------------------------------------------------
+Function: fza_fnc_getAttackProfile
```

Comment:
When putting a function name in the a folder here, we add the folder name as a prefix - so this should be `hellfireGetAttackProfile`. However, at this point the hellfire stuff is getting large enough it may be better to have it as its own PBO. I can help you with that if you'd like, and would change it from `fza_fnc_hellfireGetAttackProfile` to `fza_hellfire_fnc_getAttackProfile`